src/markdown_renderers.js

```diff
@@ -4,9 +4,7 @@ function formatIdTag(idTag) {
     // Get rid of whitespace and random characters
     idTag = idTag.toLowerCase().trim().replace(/[^\w]+/g, '-');
     // Remove dashes at the start
-    while (idTag[0] === '-') {
-        idTag = idTag.replace(idTag[0], '');
-    }
+    idTag = idTag.replace(/^[-]+|[-]+$/g, '');
```

Comment:
This covers also the "Remove trailing dashes" case :)

Reply:
:)) thank you!